docs/.eleventy.js

```diff
@@ -26,7 +26,7 @@ module.exports = function(eleventyConfig) {
   eleventyConfig.addPlugin(markdownPlugin);
 
   // Add submenu generation
-  eleventyConfig.addPlugin(pluginTOC, {tags: ['h2', 'h3'], wrapper: 'nav aria-label="Table of contents"'});
+  eleventyConfig.addPlugin(pluginTOC, {tags: ['h2', 'h3'], wrapper: 'nav aria-label="Table of contents"', wrapperClass: 'toc s-anchors s-anchors__muted'});
```

Comment:
I was wondering if moving the anchors classes here actually creates now 2 places for us to check how the `toc` is styled (this file and the `stacks-documentation.less` file). I would probably prefer to keep everything in one place as much as possible. I don't feel strongly about it though - feel free to ignore the suggestion.

Reply:
I hear you on this. This is the best solution I could come up with that avoided either a refactor of how our `toc` was rendered or including the equivelent `.s-anchors.s-anchors__muted` styling ported manually to `toc`. It's a trade-off, but I think a reasonable one.